### ISSUE TITLE
Documentation incomplete

### ISSUE THREAD

**Issue by krisavi** [cybertec-postgresql/pgwatch#571]
**Describe the bug**
Documentation is very lacking. ['Custom installation'](https://github.com/cybertec-postgresql/pgwatch/blob/master/docs/tutorial/custom_installation.md) starts with building Go applications and then there are references to files that do not exist and after that it goes to instruct using files and parameters that were used for PgWatch2. Neither code nor --help doesn't reveal what parameters should be used to set up non-docker installations.

**To Reproduce**
1. Go to ['Custom installation'](https://github.com/cybertec-postgresql/pgwatch/blob/master/docs/tutorial/custom_installation.md)
2. Try to follow or set up using it

**Expected behavior**
Documentation to have correct information.

**Additional context**

> Sample startup scripts can be found at /etc/pgwatch/startup-scripts/pgwatch.service or online [here](https://github.com/cybertec-postgresql/pgwatch/blob/master/pgwatch/startup-scripts/pgwatch.service). Note that they are OS agnostic and might need some light adjustment of paths, etc - so always test them out.

Nope, nope, nope. Just doesn't exist in those locations.

**Comment by ondar** [comment]
Yeah, I went through the docs too and for a lot of typos and remnants from pgwatch2. See https://github.com/cybertec-postgresql/pgwatch/pull/570

**Comment by krisavi** [comment]
Been trying to figure out how to configure patroni lookup for the database, but don't see any way to configure etcd parameters.
For database it asks for connectionstring, entering password plaintext into there doesn't seem like a good idea either.
